tests/basic/sharedv4_svc_test.go

```diff
@@ -309,7 +313,7 @@ func (fm *failoverMethodMaintenance) sleepBetweenFailovers() time.Duration {
 
 var _ = Describe("{Sharedv4SvcFunctional}", func() {
 	var testrailID, runID int
-	var contexts, testSv4Contexts []*scheduler.Context
+	var contexts, testSv4Contexts, testSharedV4Contexts []*scheduler.Context
```

Comment:
rename testSv4Contexts to testSv4SvcContexts?

Another thing that is a bit odd is that this file, the job everything is named as sharedv4-svc functional tests. So, this new test does not fit well in that scheme. Any thoughts on how we can clean that up?

Reply:
good point. i can move it to its own scheme. maybe even to a new file, since this file is also called sharedv4_svc test